scripts/setup-adapters.sh

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+function install_aws-sdk-cpp {
+  local NAME="aws-sdk-cpp"
+  local AWS_SDK_VERSION="1.9.96"
+
+  if [ -d "${NAME}" ]; then
+    read -p "do you want to rebuild '${NAME}'? (y/n) " confirm
+    if [[ "${confirm}" =~ ^[yy]$ ]]; then
+      rm -rf "${NAME}"
+    else
+      return 0
+    fi
+  fi
+
+  git clone --depth 1 --recurse-submodules --branch "${AWS_SDK_VERSION}" https://github.com/aws/aws-sdk-cpp.git "${NAME}"
+  mkdir "${NAME}_build"
+  cd "${NAME}_build"
```

Comment:
Got a warning here and in other similar places: (warning) Use 'cd ... || exit' or 'cd ... || return' in case cd fails.